packages/react-components/react-card/src/components/CardHeader/useCardHeaderStyles.ts

```diff
@@ -8,46 +8,33 @@ import type { CardHeaderSlots, CardHeaderState } from './CardHeader.types';
 export const cardHeaderClassNames: SlotClassNames<CardHeaderSlots> = {
   root: 'fui-CardHeader',
   image: 'fui-CardHeader__image',
-  content: 'fui-CardHeader__content',
```

Comment:
isnt this a breaking change ?

Reply:
Yep. Is there a specific action I should take, given that Card is still unstable and "alpha"?

Reply:
I'd mention that in change file so its generated in changelog as breaking change

Reply:
good question, I'd start with https://github.com/microsoft/fluentui/blob/master/rfcs/convergence/changelog-guidelines.md, but it's missing how we annotate "Breaking changes" cc @ling1726 . 

Also the json format might be a bit limiting in comparison with changesets and its markdown support, but not sure TBH

Reply:
I meant in general, since the RFC doesn't cover it. Is adding "BREAKING CHANGE" before the message the standard?

Reply:
we should bring this up on tech sync and update the RFC so we have proper style-guide + tooling. wdyt?

Reply:
Done on https://github.com/microsoft/fluentui/pull/23588